.github/workflows/check.yml

```diff
@@ -34,6 +33,7 @@ jobs:
     - name: Install dependencies
       run: poetry install -E crypto
     - name: Lint with flake8
+      if: ${{ matrix.python-version != '3.12' }}
```

Comment:
Sorry, missed it, why we are skipping  linter for 3.12?

Reply:
Yeah, I removed the check in latest commit. Seems we also need to upgrade `flake8` for python 3.12

Reply:
we have moved linting to a separate file